rl4j-examples/src/main/java/org/deeplearning4j/examples/rl4j/A3CALE.java

```diff
@@ -34,16 +34,16 @@
 public class A3CALE {
 
     public static HistoryProcessor.Configuration ALE_HP =
-            new HistoryProcessor.Configuration(
-                    4,       //History length
-                    84,      //resize width
-                    110,     //resize height
-                    84,      //crop width
-                    84,      //crop height
```

Comment:
That's the size they've been using in the original paper about this. Why not leave it like that?

Reply:
Didn't realise these were the original paper params. 
Looking at the paper I think that rl4j is doing cropping first then resize rather than resize and then crop which is what the paper requires. 

I could change the params to do that same thing though, crop while zoomed out and then scale to 84x84?

Reply:
Either way, the size of the video frame should work as long as the game is actually visible in the frame

Reply:
I didn't realize the order was wrong. Yes, please fix it! Of course we can use any resolution, it's just going to make it much slower, so let's leave it at 84x84.

Reply:
have changed these to output a 84x84 screen size for training, but the video transformation implementation is doing cropping followed by rescale so the parameters still need to be different from before.